content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -18,14 +18,14 @@ Before you can follow this guide, you will need to set up your application to be
 
 ## Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-The Prisma schema should contain the following in the `generator` block:
+Since your code is being deployed to Netlify's environment, which isn't necessarily the same as your development environment, you will need to set [`binaryTargets`](/reference/api-reference/prisma-schema-reference#binarytargets-options) in order to download the query engine that is compatible with the Netlify runtime during your build step. If you do not set this option, your deployed code will have an incorrect query engine deployed with it and will not function.
```

Comment:
I just realized again the difference for Netlify, is If I remember correctly that the build env is different than the deplyoment env actually 🙈